src/assets/styles/forms.scss

```diff
@@ -46,7 +46,6 @@ form {
 }
 
 .form-search-wrapper {
-  display: flex;
   align-items: center;
```

Comment:
można by to było zastąpić wbudowanymi klasami bootstrapa, ale może tak jest nawet bardziej optymalnie jedną klasą to ograć